packages/blade/src/_helpers/storybook/useMakeFigmaURL.ts

```diff
@@ -0,0 +1,32 @@
+import { DocsContext } from '@storybook/addon-docs';
+import { useContext } from 'react';
+
```

Comment:
this hook helps to give the figma URLs based on Theme and Color Scheme selected on Storybook's UI panel.

Why hook? Because to get access to globals I need to use `DocsContext` using React's `useHook` so had to name this prefixed with `use` 😞